components/Feature/VulnerabilitiesGrid/index.js

```diff
@@ -20,19 +20,25 @@ function createLookup() {
         .map(item => item.label)
         .concat(group.name)
     );
+
     lookup.set(name, values);
   });
 
   return lookup;
 }
 
-const VulnerabilitiesGrid = ({ resources, onUpdate }) => {
+const VulnerabilitiesGrid = ({
+  resources,
+  onUpdate,
+  initialExpandedGroups = {}
+}) => {
   const [grid, setGrid] = useState({
     assets: {},
     vulnerabilities: {},
     data: {}
   });
   const groupItems = useMemo(() => createLookup());
+  const [expandedGroups, setExpandedGroups] = useState(initialExpandedGroups);
```

Comment:
Might just not be able to see it, but what happens if I click "Open all" to open all the accordion sections? Is there something listening for that too?

Reply:
Didn't think about it! Good spot!